tests_e2e/test_suites/images.yml

```diff
@@ -57,18 +59,19 @@ image-sets:
 # '<Publisher>:<Offer>:<Sku>:<Version>'
 #
 images:
-#
```

Comment:
removed this duplicated comment... these distros are not added to 'endorsed' yet, and there is already a TODO comment there